src/libs/ajax/leonardo/providers/LeoRuntimeProvider.test.ts

```diff
@@ -85,6 +99,121 @@ describe('leoRuntimeProvider', () => {
     expect(result).toEqual([]);
   });
 
+  it('handles errorInfo call - GCP', async () => {
```

Comment:
Suggest a `describe` block to organize related tests.

```ts
  describe('errorInfo call', () => {
    it('...', async () => {
    ...
```

The `describe` block also provides a place to define a `setup` function for a group of tests.